app/src/constants/UrlLink.js

```diff
@@ -0,0 +1 @@
+export const BASE_API_URL = 'http://localhost:8080/api/'
```

Comment:
This might need to be in a property file so deploying on production, test etc won't require source code changes